index.html

```diff
@@ -1,255 +1,373 @@
-		 <div class="about">
-		   <h2 class="title3">About Myself</h2>
-		   <p class="title4">Hello I'm a software developer! I can help you build a product , 
-			feature or website Look through some of my work and experience! 
-			If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-	 
-		   <div class="social-icon2">
-			<h3 class="connect2" href="#">LET'S CONNECT</h3>
-			<ul>
-				<li><a href="https://github.com/Gardimy"><img src="Images/github.png" alt="GitHub Icon"></a></li>
-				<li><a href="https://www.linkedin.com/in/gardimy-charles-40b795212/"><img src="Images/linkedin.png" alt="Linkedin Icon"></a></li>
-				<li><a href="#"><img src="Images/Fingers icon.png" alt="Fingers Icon"></a></li>
-				<li><a href="https://twitter.com/"><img src="Images/twitter.png" alt="Twitter Icon"></a></li>
-				<li><a href="#"><img src="Images/medium.png" alt="Medium Icon"></a></li>
-			  	</ul>
-		  </div>
-			   <button class="seeproject">Get my resume</button>
-		  </div>
-		 <div class="languages-details">
-		   <div class="languages">
-			 <h3 class="languagesn">Languages</h3>
-			 <i class="arrow down"></i>
-		   </div>
-	 
-		   <ul class="languages-list">
-			 <li class="languages-Icon"><img class="Icons" src="Images/Js.icon.png" alt="JavaScript">
-				<p class="name">javascript</p></li>
-			
-			 <li class="languages-Icon"><img class="Icons" src="Images/Html.icon.png" alt="HTML">
-				<p class="name">HTML</p></li>
-			
-			 <li class="languages-Icon"><img class="Icons" src="Images/Css.icon.png" alt="CSS">
-				<p class="name">CSS</p></li>
-		   </ul>
-	 
-		   <div class="languages line">
-			 <h3 class="languagesn">Frameworks</h3>
-			 <i class="arrow right"></i>
-		   </div>
-
-		   <div class="languages line">
-			 <h3 class="languagesn">Skills</h3>
-			 <i class="arrow right"></i>
-		   </div>
-		 </div>
-		</section>
-		<section class="Myform">
-			<div class="form-container">
-			  <h2 class="contact-title">Contact me</h2>
-			  <p class="contact-text">If you have an application you are
-				 interested in developing, a feature that you need built 
-				 or a project that needs coding. I'd love to help with it </p>
-			  <div class="enter">
-				<form action="https://formspree.io/f/xbjbnvvz" method="POST">
-				  <input class="entername" type="text" 
-				  placeholder="Enter your name here" maxlength="30" required>
-				  <input class="entername" type="email" 
-				  placeholder="Enter your Email here" required>
-				  <textarea class="entermessage" name="area" cols="30" rows="6"  maxlength="500" 
-				  placeholder="Write your message here" required></textarea>
-				  <button class="get-in-touch" type="submit">Get in touch</button>
-				</form>
-			  </div>
-			</div>
-		</section>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
```

Comment:
- Kindly make sure your name is in the title as it is a good practice for devs portfolio :recycle: